packages/sdk-middleware-http/src/http.js

```diff
@@ -68,14 +68,15 @@ export default function createHttpMiddleware({
   } = {},
   fetch: fetcher,
   abortController: _abortController,
+  getAbortController
 }: HttpMiddlewareOptions): Middleware {
   if (!fetcher && typeof fetch === 'undefined')
     throw new Error(
       '`fetch` is not available. Please pass in `fetch` as an option or have it globally available.'
     )
-  if (timeout && !_abortController && typeof AbortController === 'undefined')
+  if (timeout && !getAbortController && !_abortController && typeof AbortController === 'undefined')
     throw new Error(
-      '`AbortController` is not available. Please pass in `AbortController` as an option or have it globally available when using timeout.'
+      '`AbortController` is not available. Please pass in `getAbortController` as an option or have AbortController globally available when using timeout.'
```

Comment:
This change will fail this test

https://github.com/commercetools/nodejs/blob/693f630b2c8700bcb68d2c79048b075c92ca558f/packages/sdk-middleware-http/test/http.spec.js#L34-L42